### ISSUE TITLE
[Manure] Verify MCF Equation for CBPB 

### ISSUE THREAD

**Issue by tomhuhh** [RuminantFarmSystems/RuFaS#1665]
## Expected Behavior

This issue is related to PR #1598.

The equation used for calculating methane conversion factor (MCF) for Compost Bedded Pack Barns (CBPB) should yield estimated methane emissions that align with values reported in the literature.

## Current Behavior

We are unclear about the source of the CBPB equation. The equation in the current version of the scientific document is different from what we use in the code.

## Possible Solution

Validate the reference for the CBPB scenario by comparing the methane emission estimates with literature data. Consult subject matter experts, such as April, to ensure the equation's accuracy.

## Steps to Reproduce

1. Identify the current equation used in the code for CBPB.
2. Look up the equation in the scientific document. 
3. Compare the equations.  

## Context (Environment)

This issue impacts the estimates for manure methane emissions within the CBPB manure treatment system. Accurate methane emission estimates are crucial for environmental impact assessments and regulatory compliance.

## Detailed Description

The equation currently used for calculating the methane conversion factor (MCF) for Compost Bedded Pack Barns (CBPB) differs from the one in the scientific document. We need to verify if the current equation is appropriate by comparing its methane emission estimates with values reported in the literature. This preventative verification aims to ensure the accuracy and reliability of our methane emission estimates for CBPB.

## Possible Implementation

1. Review the current equation used in the code.
2. Gather and analyze literature data on methane emissions for CBPB.
3. Consult subject matter experts, such as April, for further insights and validation.
4. Update the equation in the code if necessary to align with verified data.

## Target Deadline

As soon as possible.

**Comment by elle-andreen** [comment]
@tomhuhh Would you consider this issue closeable since the CBPB equation source has been identified (IPCC, 2019) and has been appropriately documented in the manure module scientific documentation?

**Comment by tomhuhh** [comment]
Hi Elle! Thanks for pointing this out. I’m a bit confused actually. It looks like the code still uses the IFSM method in `calculate_methane_conversion_factor`, whereas our [“RuFaS Manure Module: Bedded Pack”](https://docs.google.com/document/d/1t52i7RODnKGFW_eVlrQBaHP9g5xU3osXsedl9I3Df9c/edit?usp=sharing) document specifies fixed MCF values based on mixing and temperature. It seems we’re not yet aligned, and I believe we want to follow the documentation’s approach. Let me know what your thoughts are. Thanks!

![Image](https://github.com/user-attachments/assets/cc46600f-2221-413e-b7a1-b69d63f5909e)

**Comment by elle-andreen** [comment]
> Hi Elle! Thanks for pointing this out. I’m a bit confused actually. It looks like the code still uses the IFSM method in `calculate_methane_conversion_factor`, whereas our [“RuFaS Manure Module: Bedded Pack”](https://docs.google.com/document/d/1t52i7RODnKGFW_eVlrQBaHP9g5xU3osXsedl9I3Df9c/edit?usp=sharing) document specifies fixed MCF values based on mixing and temperature. It seems we’re not yet aligned, and I believe we want to follow the documentation’s approach. Let me know what your thoughts are. Thanks!
> 
> ![Image](https://github.com/user-attachments/assets/cc46600f-2221-413e-b7a1-b69d63f5909e)

@tomhuhh This is true, but part of issues #2307 and #2352 is to bring bedded pack MCFs into alignment with IPCC 2019/USDA Blue Book! This should be what is reflected in the documentation, the issues to bring the code into alignment were just deferred until after the manure refresh was complete

**Comment by tomhuhh** [comment]
Closing the issue as it is tracked by issue #2307
